### PR TITLE
yarn => npm

### PR DESCRIPTION
**Pull request by adrianjost**
# Issue: [Vue-66](https://ticketsystem.schul-cloud.org/browse/VUE-66)

Wait with merging it until one more discussion on monday.
## Description

- we wan't to be consistent across all repos. Therefore we switched from yarn to npm here.

### CODE REVIEW

File: deploy/Dockerfile.nuxt [hunk 1]
```diff
@@ -4,9 +4,9 @@ RUN apt-get update && \
 
 WORKDIR '/app'
 COPY ./package.json ./
-COPY ./yarn.lock ./
+COPY ./packe-lock.json ./
```
Comment (CordlessWool) [inline deploy/Dockerfile.nuxt]: you mean package-lock.json?

Comment (adrianjost) [reply deploy/Dockerfile.nuxt]: jup. typo

**Comment by adrianjost** [comment]
> yarn.lock have to removed from git repository after merge, otherwise it will synced anymore

Isn't it removed already with this pull? @CordlessWool 
image

https://github.com/schul-cloud/nuxt-client/pull/248/files#diff-8ee2343978836a779dc9f8d6b794c3b2

File: package.json [hunk 2]
```diff
@@ -97,6 +97,7 @@
 		"core-js": "2.6.5",
 		"cross-env": "^5.2.0",
 		"css-loader": "^2.1.1",
+		"csurf": "^1.10.0",
```
Comment (adrianjost) [inline package.json]: @ChrisWeissmann this is new. Can you explain why we need it?

Comment (ChrisWeissmann) [reply package.json]: Else I get an error that it is missing when trying to start it. 
```
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'csurf'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/chris/Developer/nuxt-client/src/legacy-client/controllers/oauth2.js:4:14)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/chris/Developer/nuxt-client/src/legacy-client/controllers/index.js:46:23)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at setLegacyControllers (/Users/chris/Developer/nuxt-client/src/server/index.js:100:5)
    at Object.<anonymous> (/Users/chris/Developer/nuxt-client/src/server/index.js:128:2)
```

Comment (ChrisWeissmann) [reply package.json]: I think you are right. There was an error on my local machine. I will remove it again.
Sorry 😬

**Pull request merged**